SU-SOE/stanford_soe_helper
# SOE-3660: Adding scss/css changes for the Testimonial injector

# READY FOR REVIEW

# Summary
- Addressed styling bugs for Testimonials

# Needed By (Date)
- End of sprint

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Turn off this injector: `admin/config/development/css-injector/edit/91`
3. `drush cc all`
4. On this page, `catalog-patterns/blocks-and-beans/testimonial-blocks`, make sure that it looks good when the injector is off.
5. Check all breakpoints in the browser.

# Affected Projects or Products
- SoE
- stanford_soe_helper

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-3660

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
